(DOCSP 30002): Flutter FTS (#2847)
mongodb/docs-realm: source/sdk/flutter/realm-database/read-and-write-data.txt

.. literalinclude:: /examples/generated/flutter/read_write_data_test.snippet.filter.dart
   :language: dart

.. _flutter-fts:

Filter with Full-Text Search
~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> You can use Realm Query Language (RQL) to query on properties that have
> :ref:`Full-Text Search Indexes <flutter-fts-annotation>` (FTS) on them. 

Reviewer: ```suggestion
You can use Realm Query Language (RQL) to query on properties that use
:ref:`Full-Text Search Index <flutter-fts-annotation>` annotations. 
```
